core-android/src/main/java/com/uber/sdk/android/core/auth/LoginManager.java

```diff
@@ -270,11 +286,42 @@ public LoginManager setRedirectForAuthorizationCode(boolean redirectForAuthoriza
      * update Uber app instead.
      *
      * @return true if redirect by default, otherwise false
+     * @deprecated, See Authorization Migration guide https://github.com/uber/rides-android-sdk#authentication-migration-version-08-and-above
      */
+    @Deprecated
     public boolean isRedirectForAuthorizationCode() {
         return redirectForAuthorizationCode;
     }
 
+
+    /**
+     * Enable the use of the Authorization Code Flow
+     * (https://developer.uber.com/docs/authentication#section-step-one-authorize) instead of an
+     * installation prompt for the Uber app as a login fallback mechanism.
+     *
+     * Requires that the app's backend system is configured to support this flow and the redirect
+     * URI is pointed correctly.
+     *
+     * @param authCodeFlowEnabled true for use of auth code flow, false to fallback to Uber app
+     * installation
+     * @return this instace of {@link LoginManager}
+     */
+    public LoginManager setAuthCodeFlowEnabled(boolean authCodeFlowEnabled) {
```

Comment:
We previously were using setRedirectForAuthorizationCode() to indicate that it shouldn't redirect to the Playstore and should instead use auth code flow. I've now replaced that with a new method here that's more explicit. We can also use the the state of the old one being set to give us an indication that the developer had a previous implementation and we should treat it carefully in upgrading them to chrometabs. See IsInLegacyMode() below.